driver-core/src/test/java/com/datastax/driver/core/ProtocolBetaVersionTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import com.datastax.driver.core.exceptions.NoHostAvailableException;
+import com.datastax.driver.core.exceptions.UnsupportedProtocolVersionException;
+import org.testng.annotations.Test;
+
+import static com.datastax.driver.core.ProtocolVersion.V4;
+import static com.datastax.driver.core.ProtocolVersion.V5;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+/**
+ * Tests for the new USE_BETA flag introduced in protocol v5
+ * and Cassandra 3.10.
+ */
+@CCMConfig(createCluster = false, version = "3.10")
```

Comment:
This test requires  version 3.10 _precisely_, just as `ProtocolV1Test` requires 1.2.19.

Until 3.10 is released, here is what you should do to make the tests pass:

```
cd /path/to/cassandra/local/git/repo
git fetch
git checkout 12142-trunk
ant build
cd ~/.ccm/repository
ln -s /path/to/cassandra/local/git/repo 3.10
```

Note that you will have to delete the symlink after 3.10 is released.